.github/workflows/ci.yml

```diff
@@ -87,16 +98,36 @@ jobs:
       run: python setup.py sdist
 
     - name: Install requirements
-      run: python -m pip install enum34 pathlib2 importlib_resources attrs hepunits tabulate --target src
-
-    - name: Move enum to prevent name clash
-      run: mv src/enum src/enum34
+      run: python -m pip install attrs hepunits tabulate importlib_resources --target src
 
     - name: Make ZipApp
-      run: python -m zipapp -p "/usr/bin/env python" -o ../particle.pyz .
+      run: python -m zipapp -c -p "/usr/bin/env python3" -m "particle.__main__:main" -o ../particle.pyz .
```

Comment:
Just for my understanding - does this mean that the ZipApp will only be available for Python 3?

Reply:
Yes. It lets us drop hacks and workarounds, and drop some packages to make it smaller. Importlib resources was requiring some new dependency only in Python 2 making it harder. You don't need to use python3 for the rest of your work, you just need installed on your computer.

Reply:
OK!